packages/nextjs/pages/index.tsx

```diff
@@ -25,7 +26,31 @@ const Home: NextPage = () => {
           Edit your smart contract <code className="italic bg-secondary text-base font-bold">YourContract.sol</code> in{" "}
           <code className="italic bg-secondary text-base font-bold">packages/hardhat/contracts</code>
         </p>
-
+        {/* TODO REMOVE THIS BEFORE MERGING */}
```

Comment:
Added this for testing, we can remove it before merging. 

please let me know if I should remove it now itself🙌

Reply:
This is cool, let's keep this conversation unresolved so we don't forget :)